.gitlab/ci/on-push.yml

```diff
@@ -25,21 +21,19 @@ trigger-private-build:
     - python3 Utils/get_private_build_status.py --github-token $GITHUB_TOKEN
 
 create-instances:
-  extends:
+  extends: 
     - .default-job-settings
   rules:
     - if: '$CI_PIPELINE_SOURCE =~ /^(push)$/'
     - if: '$NIGHTLY'
     - if: '$INSTANCE_TESTS'
   cache:
     policy: pull-push
-    when: always
   needs: []
   stage: create-instances
```

Comment:
this doesn't run anymore on DEMISTO_SDK_NIGHTLY right? remove its logic from this job then.